Doc/library/csv.rst

```diff
@@ -212,11 +212,9 @@ The :mod:`csv` module defines the following classes:
    A short usage example::
 
        import csv
-
```

Comment:
why remove this?  empty lines let example code breathe.

Reply:
Because it is the case for 10 other code samples in the page, so again for consistency.

Reply:
You missed or ignored my point about the value of consistency.

Reply:
I also prefer an empty line after `import` statements, but ten other code samples in the page don’t use them. So we should either remove the empty line in this code sample or add empty lines in the ten other code samples. What do you prefer?

Reply:
_Consistency_ of _aesthetic_ details is not a goal of the Python documentation!

Lack of consistency in the signature of related functions can be a problem for end users.
Having two different ways of writing tests for the same module can be a problem for contributors.

But simple spaces or empty lines are not a problem, and changing them does not really improve things.
There is no need here to remove some or add some.  The code is not broken or misleading or hard to change.
So, no problem, no change needed.

(Even for non-aesthetic changes, because of the long history of the CPython codebase and the number of contributors, there isn’t a 100% unified code style, doc writing style, testing style, etc.  Smaller, newer projects can use policies and tools to change things in all their code or docs, but not CPython.